machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc

```diff
@@ -15,6 +15,18 @@ create your cluster. If you do not have these files, you must obtain them by
 following the instructions in the
 xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[installation procedure].
 
+[IMPORTANT]
+===
+When adding additional compute machines to your {product-title} cluster, you should use the
+{op-system} boot media that matches the same minor version that was used to originally
+install the {product-title} cluster.  For example, if you originally installed
+{product-title} 4.4, you should add additional compute machines using {op-system} 4.4
+boot media.
+
+It is not supported to add additional compute machines to your cluster using newer
```

Comment:
Adding additional compute machines to your cluster using newer {op-system} boot media is not supported.
```suggestion
It is not supported to add additional compute machines to your cluster using newer
```